drivers/flash/flash_mspi_nor_quirks.h

```diff
@@ -63,12 +63,6 @@ static inline int mxicy_mx25r_post_switch_mode(const struct device *dev)
 		return 0;
 	}
 
-	/* Wait for previous write to finish */
-	rc = wait_until_ready(dev, K_USEC(1));
-	if (rc < 0) {
-		return rc;
-	}
-
```

Comment:
Wrong commit? This change is not mentioned in the commit description.

Reply:
This waiting here is not needed because `quad_enable_set()` waits after writing the status register. But I can mention this in the commit message so there's no doubt.